File: jython/uuid_sample.py
# filename: uuid_sample.py
from java.lang import System
from org.jruby.embed import ScriptingContainer
c = ScriptingContainer()
paths = [System.getProperty("user.dir") + "/jython/lib/jruby/1.8/gems/uuidtools-2.1.1/lib"]
c.setLoadPaths(paths)
c.runScriptlet("require 'rubygems'; require 'uuidtools'")
print c.runScriptlet("UUIDTools::UUID.timestamp_create")

File: scala/uuid_sample.scala
// filename: uuid_sample.scala
import scala.collection.JavaConversions._
import scala.collection.mutable.ListBuffer
import org.jruby.embed.ScriptingContainer

val c = new ScriptingContainer
val dirs = List(System.getProperty("user.dir") + "/scala/lib/jruby/1.8/gems/uuidtools-2.1.1/lib")
val paths: java.util.List[String] = ListBuffer(dirs: _*)
c.setLoadPaths(paths)
c.runScriptlet("require 'rubygems'; require 'uuidtools'")
println(c.runScriptlet("UUIDTools::UUID.timestamp_create"))
